src/github/entities/allowed-organization.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { OrganizationStatus } from 'src/common/constants';
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity } from 'typeorm';
+
+@Entity()
+export class AllowedOrganization extends Base {
+  @Column({ type: 'varchar', length: 255, default: 'github' })
+  host: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true })
+  organizationName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminUserName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminEmail: string;
+
+  @Column('text', { array: true, default: [] })
+  activeUsers: string[];
+
+  @Column('text', { array: true, default: [] })
+  inactiveUsers: string[];
```

Comment:
PullProofAI

The usage of arrays for `activeUsers` and `inactiveUsers` is appropriate for storing multiple values. Ensure that these arrays are properly handled during CRUD operations to prevent data inconsistencies.

```diff
+ // Ensure proper handling of arrays activeUsers and inactiveUsers during CRUD operations
```



<!-- This is an auto-generated comment by PullProofAI -->